lms/tasks/email_digests.py

```diff
@@ -78,24 +78,23 @@ def send_instructor_email_digests(
     The email digests will cover activity that occurred in the time period
     described by the `updated_after` and `updated_before` arguments.
 
-    :param h_userids: the h_userid's of the instructors to email
-    :param updated_after: the beginning of the time period as an ISO 8601 format string
-    :param updated_before: the end of the time period as an ISO 8601 format string
-
+    :param h_userids: h_userid's of the instructors to email
+    :param updated_after: the beginning of the time period as an ISO 8601
+        format string
+    :param updated_before: the end of the time period as an ISO 8601 format
+        string
     :param override_to_email: send all the emails to this email address instead
         of the users' email addresses (this is for test purposes)
     """
-    # Caution: datetime.fromisoformat() doesn't support all ISO 8601 strings!
-    # This only works for the subset of ISO 8601 produced by datetime.isoformat().
-    updated_after = datetime.fromisoformat(updated_after)
-    updated_before = datetime.fromisoformat(updated_before)
 
     with app.request_context() as request:  # pylint:disable=no-member
         with request.tm:
-            digest_service = request.find_service(DigestService)
-            digest_service.send_instructor_email_digests(
-                h_userids,
-                updated_after,
-                updated_before,
+            request.find_service(DigestService).send_emails(
+                audience=h_userids,
+                # Caution: `datetime.fromisoformat()` doesn't support all ISO
+                # 8601 strings! This only works for the subset of ISO 8601
+                # produced by `datetime.isoformat()`.
+                updated_after=datetime.fromisoformat(updated_after),
+                updated_before=datetime.fromisoformat(updated_before),
                 override_to_email=override_to_email,
```

Comment:
The functional change here is because I renamed `send_instructor_email_digests` -> `send_emails`.

The rest was just some tidying while I was here.